_analyzers/token-filters/limit.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: default
+title: Limit
+parent: Token filters
+nav_order: 250
+---
+
+# Limit token filter
+
+The `limit` token filter is used to limit the number of tokens that are passed through the analysis chain.
+
+## Parameters
+
+The `limit` token filter can be configured with the following parameters.
+
+Parameter | Required/Optional | Data type | Description
+:--- | :--- | :--- | :--- 
+`max_token_count` | Optional | Integer | The maximum number of tokens to be generated. Default is `1`.
+`consume_all_tokens` | Optional | Boolean | (Expect level setting) Use all tokens from tokenizer, even if result exceeds `max_token_count`. When this parameter is set, the output still only contains the number of tokens specified in `max_token_count`. However, all tokens generated by the tokenizer are processed. Default is `false`.
+
```

Comment:
Line 19: What does "(Expect level setting)" mean?

Reply:
Expert 😄 Thanks for the catch.

Reply:
YW, but should be "Expert-level setting" in that case 😄